papaya-integration-tests/src/test/java/pl/edu/agh/papaya/tests/util/keycloak/KeycloakConfigurationProvider.java

```diff
@@ -0,0 +1,39 @@
+package pl.edu.agh.papaya.tests.util.keycloak;
+
+import org.keycloak.adapters.springboot.KeycloakSpringBootProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+import pl.edu.agh.papaya.security.KeycloakClientConfiguration;
+
+@Component
+public class KeycloakConfigurationProvider {
+
+    @Autowired
+    private KeycloakSupervisor keycloakSupervisor;
```

Comment:
Nie pamiętam, czy tutaj dotarła inicjatywa "pola final bez `@Autowired`, `@RequiredArgsConstructor` na klasie"? Tu jest lombok w ogóle? Nie ma? Ja mogłam próbować to zmienić, ale coś nie zadziałało natychmiast i się poddałam, bo czytoważnechybanieważne